pkg/plugins/repo/models.go

```diff
@@ -1,83 +1,27 @@
 package repo
 
-import (
-	"archive/zip"
-	"fmt"
-)
+import "archive/zip"
 
 type PluginArchive struct {
 	File *zip.ReadCloser
 }
 
-type PluginDownloadOptions struct {
-	PluginZipURL string
-	Version      string
-	Checksum     string
+type PluginArchiveInfo struct {
+	URL           string
+	PluginVersion string
```

Comment:
nit: If you have removed `Plugin` from the URL var name, why adding it to the version?

Reply:
Heh yeah I added it in case someone though of `Version` must be the version of the archive / and not the plugin itself? 🤔 Maybe that's irrational though so can change if you think!

Reply:
the version of the archive is the same than the version of the plugin, isn't it?

Reply:
My brain was more thinking does archive file == plugin, and can we think about them in the same context.
Because `PluginArchiveInfo.URL`, `PluginArchiveInfo.Checksum` is obvious when thinking about it from a file resource perspective, and `PluginArchiveInfo.PluginVersion` removes any potential confusion rather than just `Version`.

 I think it's just Friday causing my brain to overthink.

How about just:

```go
PluginArchiveInfo struct {
	URL           string
	Version       string
        Checksum      string
}
```

Reply:
yes, yes that makes sense, I was just double checking there is yet another version number I didn't know about :D